packages/css/src/components/mega-menu/mega-menu.scss

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+.amsterdam-mega-menu {
+  padding-block: var(--amsterdam-mega-menu-padding-block);
+}
+
+.amsterdam-mega-menu__list-wrapper {
+  column-gap: var(--amsterdam-mega-menu-list-wrapper-column-gap);
+  columns: var(--amsterdam-mega-menu-list-wrapper-column-width, 20rem) auto;
```

Comment:
We generally don't define fallbacks

Reply:
We can also use `column-width` if the longhand sets `column-count` to its default of `auto` anyway.

Reply:
The fallback is defined to prevent `columns:  auto` if the slim chance exists the variable is not defined.